playbooks/de-populate-consul.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: "Update keys in consul"
+  hosts: consul
+  run_once: true
+  become: true
+  gather_facts: false
+  tags:
+    - consul-kv
+  roles:
+    - role: de-populate-consul
+      kv:
+        - {key: "configs/{{environment_name}}/amqp/uri", value: "{{amqp_de_uri}}"}
+        - {key: "configs/{{environment_name}}/amqp/exchange/name", value: "{{amqp.de.name}}"}
+        - {key: "configs/{{environment_name}}/amqp/exchange/type", value: "{{amqp.de.type}}"}
+        - {key: "configs/{{environment_name}}/apps/de-callback-uri", value: "{{apps.base}}/callbacks/de-job"}
+        - {key: "configs/{{environment_name}}/de-db/uri", value: "postgresql://{{db_user}}:{{db_password}}@{{db_host}}:{{db_port}}/{{db_name}}?sslmode=disable"}
+        - {key: "configs/{{environment_name}}/irods/user", value: "{{irods.user}}"}
+        - {key: "configs/{{environment_name}}/irods/pass", value: "{{irods.password}}"}
```

Comment:
I don't think we need to worry about it too much. We're the only ones who are going to be running these playbooks with our configuration settings for the time being.